### PR TITLE
Feature/convert reversal index/685

### PR DESCRIPTION
**Pull request by aidanpscott**

### CODE REVIEW

File: convert/convertReverseIndex.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    homonym_index?: number;
+}
+
+function convertReverseIndex(dataDir: string, language: string, alphabet: string[]): void {
+    const indexFilePath = path.join(dataDir, 'reversal', `lexicon-${language}.idx`);
+    const outputDir = path.join('static', 'reversal', 'language', language);
+
+    if (!existsSync(indexFilePath)) {
+        console.warn(`No reversal index found for language: ${language}`);
+        return;
```
Comment (chrisvire) [inline convert/convertReverseIndex.ts:18]: This show throw an exception and stop the convert process, not just return.

**Review by chrisvire — CHANGES_REQUESTED** [review]
This approach doesn't work for words that start with accented characters.  Here is a project:

https://drive.google.com/file/d/1fFksgphKDmZ1BsaepeGBrTqyCABoyJlf/view?usp=drive_link

There words were not included in your statc/reversal/language/fr/*.json files:

à côté  1296
à l'intérieur   81
à l’aise        1149
à la maison     477
à la place de (emp. à l’arabe)  901

œil     1178
œuf     727

In the native app, they were under a and o.

**Review by aidanpscott — COMMENTED** [review]
Changes have been made

**Review by chrisvire — APPROVED** [review]
👍

**Pull request merged**